doc-src/SASS_REFERENCE.md

```diff
@@ -2534,6 +2534,53 @@ passed block are related to the other styles around where the block is defined.
       }
     }
 
+### Including a Mixin Dynamically
+
+There are times when it is convenient to call a mixin based on the
+result of a SassScript expression.
+
+For example, a library might want to delegate some styling to the
+application by allowing the application to register a mixin that the
+library would include where appropriate. Sometimes a content block is
```

Comment:
This use-case won't work well in a module world—the application's mixin won't be in scope in the library context where the dynamic include is invoked. To allow users to "pass in" mixins, we'd probably need a combination of #1582 and #871 so that users can do something like:

```scss
@use "fancy";

@include fancy.register-style("background") using ($color) {
  background: $color url(logo.png);
}
```

If we think users are likely to use this functionality as a means to pass mixin definitions upstream to libraries, we should discuss whether it makes sense to land it right now. I'm very worried about adding features that will make it hard or impossible for users to upgrade to 4.0 without breaking functionality.